### DISCUSSION TITLE
dracula theme

### DISCUSSION THREAD

**Discussion by Aeilxs** [Manjaro-Sway/manjaro-sway#879] [Q&A]
hi, when you use `manjaro-sway-theme` and pick dracula, it install everything nice and clean and smooth perfect BUT the kvantum theme is not included in the downloading it seems so everytime you reboot it open kvantum (while keeping old kvantum theme i assume)

in `$HOME/.config/sway/definitions.d/theme.conf` line
set $kvantum-theme Dracula-purple-solid

i cloned the kvantum theme and installed it im fine but i'd be really happy to get some guidance about how can i try to fix it for everyone :') thanks for this manjaro distro btw it's my first tile manager and i love it

**Comment by robin-thoene** [comment]
Hey I just tested it on my machine and I can reproduce this.

I would suggest creating a bug issue for that. It should be possible to fix this in the current theme code, as for other themes the kvantum theme is installed and found automatically.

I will try to fix that, thanks for reporting.
